docs/blog/2019-04-19-gatsby-why-we-write/index.md

```diff
@@ -0,0 +1,119 @@
+
+The challenge, is of course _how do we write all of that content?_
+
+## Principle 4: Leverage ourselves through collaboration.
+
+Some examples:
+
+- We strive to appear on podcasts, because of the huge reward / time effort, as well as because podcasts build on an existing community/listener base.
+
+- We encouraging the community to give conference talks, write on their own blogs, and on the Gatsby blog. Individuals can establish themselves publicly as talented developers and Gatsby experts, while companies using Gatsby can publicly showcase their efforts for both internal and recruiting benefit.
+
+- Jason's livestreams are fundamentally collaborative — inviting one person who is an expert in a different technology for cross-pollination.
+
+- We maintain close relationships with educators who create comprehensive, complementary content, and provide encouragement to first-time users that _you can do this!_.
+
+If you're doing interesting, innovative things with Gatsby, especially in a professional context, we'd love you to write blog posts, tutorials, and documentation about what you're doing!
+
+## Principle 5: Content is not just individual pieces of work, it's compilations.
+
+If content were music, it wouldn't consist of just "songs" -- it would also include "playlists", "albums", "discographies", "soundtracks", and "multi-box sets".
+
+On the one hand, there is _so much left to write_. On the other hand, because we've already been writing so much for so long, we have SO MUCH CONTENT that a key challenge becomes organizing that content in useful ways that provide comprehensive treatments of particular topics.
+
+In the docs, things are structured hierarchically, so usually content is mostly adjacent to related content on the left sidebar.
+
+But outside the docs, that's far from the case. We have blog posts, [using-\* example sites / guides](https://github.com/gatsbyjs/gatsby/tree/master/examples), [starters](/starters/), conference talks, podcast appearances, tweets, and whatever Gatsby community members dream up and publish!
+
+**It's a constant challenge to organize these in ways that are useful and obvious for the end-user**. We're always attacking this in five or ten different ways -- discoverability via search, cross-linking, hierarchy, navbars and breadcrumbs, turning conference talks into blog posts and blog posts into docs. 
+
+Our goal is to make it easy for the user to gather relevant context, know where to look when they get stuck, and feel confident that they can make it if they keep on reading, learning, and trying.
+
+## Principle 6: make it easy for folks to become engrossed with Gatsby
+
+Gatsby is the future of the web. Our goal is to let people dive _as far deep into it as they want, and encourage them by providing them more and more to discover._
```

Comment:
> Gatsby is the future of the web.

This is asserted again towards the end, which seems fine...but I wonder if we could differentiate the two a bit. It could lead with something like:

> Building sites with Gatsby is the way of the future.

Reply:
Good point! Changed.